atlasdb-timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TimeLockServerIntegrationTest.java

```diff
@@ -53,8 +53,6 @@
 import io.dropwizard.testing.ResourceHelpers;
 import io.dropwizard.testing.junit.DropwizardAppRule;
 
-@Ignore("Observed ConcurrentModificationException-related flakes (e.g. build #611 on internal CircleCI)."
-        + "Fixed in atomix/copycat#231, but not part of Copycat 1.1.4 which we use.")
```

Comment:
I wanted to see the fastforward tests pass (I knew it will be flaky), but I see we get a `ConcurrentModificationException` right in the test setup. Ignoring again.